src/EditorFeatures/Test2/IntelliSense/CSharpSignatureHelpCommandHandlerTests.vb

```diff
@@ -286,13 +286,18 @@ class C
     void M(int i, int j) { }
     void M(int i, int j, int k) { }
 }
-]]></Document>)
+]]></Document>, showCompletionInArgumentLists:=showCompletionInArgumentLists)
 
                 state.SendTypeChars("(")
                 Await state.AssertSignatureHelpSession()
                 Await state.AssertSelectedSignatureHelpItem("void C.M()")
                 Assert.Equal(4, state.GetSignatureHelpItems().Count)
 
+                If showCompletionInArgumentLists Then
```

Comment:
@CyrusNajmabadi A couple tests changed in this way. Basically, now that sighelp _and_ completion show up at the same time, completion steals keyboard focus and these tests weren't expecting that.

Reply:
oh... yeah.  that's why i hate this feature :D